sorc/bufr_edtbfr.fd/edtbfr.f

```diff
@@ -227,16 +227,37 @@
 C     were updated based on those specified in this flag file entry
 C     (currently only mesonet and TAMDAR aircraft reports have a missing
 C     WMO bulletin header). These no longer match.
-C 2020-08-20  J. DONG   ADDED SETBMISS CALL TO SET BMISS TO 10E8 AND
-C     CHANGE THE CODE TO FIX FLOATING INVALID ERROR.
+C 2017-11-27  D. A. Keyser --  Recognizes new GOES-16 derived motion
+C     wind types in message type 005 (IR/long-wave in subtype 030, WV
+C     imager/deep-layer in subtype 031, visible in subtype 032, WV
+C     imager/cloud-top in subtype 034 and IR/short-wave in subtype 039).
+C 2018-02-21  D. A. KEYSER --  Modified to handle BUFR-feed upper-air
+C     reports in tanks b002/xx101-xx105:
+C        - Output message length increased from 10K (default) TO 200K
+C          via call to BUFRLIB subroutine MAXOUT.
+C        - Maximum number of data values in an uncompressed BUFR subset
+C          (MAXSS) increased from 80K (default) to 300K via use of
+C          BUFRLIB function ISETPRM (note: must link to dynamic
+C          allocation version of BUFRLIB).
+C        - Subroutine APPLY:
+C           - Increased arrays holding level data from 255 to 9000 (to
+C             account for high vertical resolution in many reports now
+C             in these tanks).
+C           - Reads mnemonic "VSIGX" from these tanks rather than "VSIG"
+C             which is only in TAC-feed tanks, and then translates its
+C             value into "VSIG" value associated with "SURF", "MAND",
+C             "SIGT" and "SIGW" levels (for editing Q.M. of levels by
+C             their vertical significance qualifier).
+C           - Increased amount of temporary diagnostic print.
+C
```

Comment:
I would go ahead and update the dates for the documentation to be current. The dates should reflect when these features became available for use in operations. For the items that Dennis worked on, you may keep his name but perhaps change to something like this: 
2022-06-17  D. A. KYESER / C. HILL -- Modified to handle ...

There are other areas of these code documentation entries that reflect the dates when previous work was done; however, since the features are not yet available in operations, I suggest moving those dates to reflect a more reasonable projection for when this upgrade will be released.

Reply:
The dates that Dennis K. initially entered with his code updates represent times at which developed code was completed without foresight of the timing of their eventual implementation to operations.  Estimates for the timing of operational implementation are ultimately reflected in the Release Notes.  To change the dates Dennis and I entered throughout the code would erase the timeline of incremental development.